website/aggregated_questions.qmd

```diff
@@ -0,0 +1,251 @@
+
+[question in general needs to be more specific, but hopefully the idea is communicated] Will synthetic DNA or gene circuits be used in human tissues for drug delivery or therapeutic purposes by 20XX?
+
+In what year will gene-editing therapies [scope of therapy and age-of-application required] for muscular dystrophy be available in over XX% of countries?
+
+By what year will the global market for gene-editing therapies surpass XX billion [possibly difficult to measure]?
+
+What percentage of all newborns in OECD countries [country designation required] will undergo polygenic risk screening for disease [can possibly be broken down by disease] by 20XX?
+
+In what year will gene-editing therapies for age-related cognitive decline [OR aging in general] be approved for use in humans?
+
+What percentage of the global population [measurement details required] will have access to gene-editing therapies for non-medical enhancements by 20XX?
+
+By what year will more than XX% of global fertility clinics [measurement details required] offer polygenic embryo screening for traits other than treatment [scope of treatment required]?
+
+What percentage of genetic engineering research funding will be dedicated to gene-editing therapies for metabolic diseases [OR other targets, scope of targets required] by 20XX?
+
+In what year will gene-editing for bone regeneration become a clinical reality [scope of clinical reality required]?
+
+What percentage of the global healthcare budget [measurement details required] will be spent on gene-editing therapies and polygenic screening by 20XX?
+
+By what year will the first CRISPR-based [OR other method] therapy to enhance muscle strength in humans complete a Phase X [scope of trials required] clinical trial?
+
+Will a gene-editing technique for increasing human lifespan enter Phase X [scope of trials required] clinical trials by 20XX?
+
+By 20XX, will there be a commercially available pre-implantation genetic diagnostic test that can screen for over XXX traits [scope of traits required]?
+
+When will a gene therapy aimed [definition of "aimed" required, can be unintentional enhancement release] at enhancing cognitive functions, such as memory or learning speed, be approved for use in any country?
+
+By what year will over 1000 babies born globally [measurement details required] have undergone polygenic screening for enhancement purposes before implantation?
+
+Will there be a breakthrough [definition of breakthrough required] in gene-editing accuracy, achieving a XX.XX% success rate [measurement details required] with no off-target effects [measurement details required], by 20XX?
+
+By 20XX, will scientists have developed a safe, reversible [scope of safe and reversible required] gene-editing system for use in human embryos?
+
+When will the first clinical trial using gene therapy to enhance sensory capabilities [scope of sensory capabilities required e.g., vision, hearing] in humans be initiated?
+
```

Comment:
if above ordinary capability is meant, then perhaps qualify as: "enhance sensory capabilities beyond a 90th percentile human, or to add fundamentally new sensory capabilities" (e.g. code for artificial opsins)
but first clinical trials may add such capabilities as a stand-in for patients with missing capabilities, e.g. an artificial opsin for people with dichromacy
also may need to rule out trivialities like https://en.wikipedia.org/wiki/OR6A2 — "using gene therapy to make persisting alterations to sensory capabilities"?